### ISSUE TITLE
still on going?

### ISSUE THREAD

**Issue by andyyuan78** [fictivelogic/cuRBM#1]

**Comment by fictivelogic** [comment]
Not exactly. I have been preoccupied by work (mostly FPGAs/VHDL) for the past few months and have not had time to come back to this project.  I may still come back to it at some point, but I'm not sure when.

**Comment by kili-ilo** [comment]
@fictivelogic :angry: :-1: boooooooooo
